sdk/digitaltwins/Azure.DigitalTwins.Core/src/Models/DigitalTwinMetadataJsonConverter.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace Azure.DigitalTwins.Core
+{
+#pragma warning disable AZC0014 // Avoid using banned types in public API
+
+    /// <summary>
+    /// JSON converter to make it easier to deserialize a <see cref="BasicDigitalTwin"/>.
+    /// </summary>
+    public class DigitalTwinMetadataJsonConverter : JsonConverter<DigitalTwinMetadata>
```

Comment:
Are there additional tests that need to be added for this? Maybe they are already covered in some other scenarios?

Reply:
Yes, but for now we're evaluating the code changer. I have updated the sample to use it, however.